src/OpenLoco/Localisation/StringManager.cpp

```diff
@@ -593,8 +593,7 @@ namespace OpenLoco::StringManager
 
             // !!! TODO: original code is prone to buffer overflow.
             buffer = strncpy(buffer, sourceStr, kUserStringSize);
-            buffer += strlen(sourceStr);
-            *buffer = '\0';
+            buffer[kUserStringSize - 1] = '\0';
```

Comment:
This fix has unintended consequences for displaying user strings. Consider this station label on master:
![Screenshot](https://user-images.githubusercontent.com/604665/190922987-a1801c2d-3653-420e-a527-5db58ed1f3c3.png)

Versus this PR:
![Screenshot (1)](https://user-images.githubusercontent.com/604665/190922986-6b5a6172-ebdc-4306-b44a-eb2bd32e3fd1.png)

Suggest we revert this change and let the #1641 wrap string PR address the issue (cc @duncanspumpkin)

Reply:
> This fix has unintended consequences for displaying user strings. Consider this station label on master: ![Screenshot](https://user-images.githubusercontent.com/604665/190922987-a1801c2d-3653-420e-a527-5db58ed1f3c3.png)
> 
> Versus this PR: ![Screenshot (1)](https://user-images.githubusercontent.com/604665/190922986-6b5a6172-ebdc-4306-b44a-eb2bd32e3fd1.png)
> 
> Suggest we revert this change and let the #1641 wrap string PR address the issue (cc @duncanspumpkin)

yes sorry you still need to increment the buffer by the strlen. I've got a better fix to that bit in the works. Revert it in this PR.